src/overlays/actors/ovl_Bg_Jya_1flift/z_bg_jya_1flift.c

```diff
@@ -9,7 +15,19 @@ void BgJya1flift_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void BgJya1flift_Update(Actor* thisx, GlobalContext* globalCtx);
 void BgJya1flift_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-/*
+void BgJya1flift_InitDynapoly(BgJya1flift* this, GlobalContext* globalCtx, void* arg2, s32 arg3);
```

Comment:
Functions should generally only be forward declared if it's necessary (ie if it's used before its actual declaration in the file), so for example the init dynapoly function here doesn't have to be.